L1Trigger/L1TMuonOverlap/src/XMLConfigWriter.cc

```diff
@@ -36,6 +38,25 @@ XERCES_CPP_NAMESPACE_USE
 #endif
 //
 
+namespace {
+ unsigned int eta2Bits(unsigned int eta) {
```

Comment:
actually, I think we are using C++14 by now, so you can also use `0b1'0000'0000`